store/storage/pebbledb/db.go

```diff
@@ -175,29 +185,6 @@ func (db *Database) Get(storeKey string, targetVersion uint64, key []byte) ([]by
 	return nil, nil
 }
 
-func (db *Database) ApplyChangeset(version uint64, cs *store.Changeset) error {
```

Comment:
could you explain why this was removed?

Reply:
it is moved to store to avoid the duplication ( the reason of abstraction layer in the storage)